### PR TITLE
Add JsonAdaptedTuitionClassTest and update UG to notify users that fi…

### LINKED ISSUE #274: Make it clear in the UG that the size specified is the class size
And make it clear that students are filtered in the students in the classes

### PR DESCRIPTION
**Pull request by Ongzl**
…ltering students will not affect class size only what is displayed
Resolves #274

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -259,6 +264,7 @@ Examples:
 
 Finds students whose `NAME` contain any of the given keywords.  
 Note that you have to run `list` to display all the students again.
+Note `Students` list in the `classes` tab will only show students found, `size` of the class will still remain constant.
```
Comment (s7u4rt99) [inline docs/UserGuide.md:267]: Maybe u can explain what constant means here. It may be slightly ambiguous. Maybe u can say `remains unchanged despite the filtering` or `is not affected by the filtering`

File: docs/UserGuide.md [hunk 2]
```diff
@@ -338,6 +344,7 @@ Examples:
 
 Finds students whose `TAG`s contain any of the given keywords.  
 Note that you have to run `list` to display all the students again.
+Note `Students` list in the `classes` tab will only show students found, `size` of the class will still remain constant.
```
Comment (s7u4rt99) [inline docs/UserGuide.md:347]: Same here as the above comment

File: src/test/java/seedu/address/storage/JsonAdaptedTuitionClassTest.java [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    private static final List<String> VALID_EMPTY_STUDENT_NAME_LIST = new ArrayList<>();
+
+    @Test
+    public void toModelType_validTuitionClassDetails_returnsPerson() throws Exception {
+        JsonAdaptedTuitionClass tuitionClass = new JsonAdaptedTuitionClass(IB_PHYSICS);
+        assertEquals(IB_PHYSICS, tuitionClass.toModelType());
+    }
+
+    @Test
+    public void toModelType_nullStudentList_success() throws Exception {
```
Comment (s7u4rt99) [inline src/test/java/seedu/address/storage/JsonAdaptedTuitionClassTest.java:41]: Should the method name be toModelType_emptyStudentList_success instead? Should there be another test where the studentList is the value `null` too?

**Review by s7u4rt99 — COMMENTED** [review]
Other than these small issues, LGTM

**Comment by softmagnet** [comment]
LGTM!

**Comment by s7u4rt99** [comment]
LGTM
